challenges/largest-product/largest_product.py

```diff
@@ -1,33 +1,39 @@
 
 # Inputs
-counter = 0
-new_list = [0]
-answer = 0
-
 test_list = [[1, 2], [2, 3], [3, 4], [4, 5]]
 
 
-def sum_inner_lists():
-    """
-    sums inner lists from existing list and returns a new list
+class LargestProduct:
     """
-    # new_list = [0]
-    # counter = 0
-    for i in test_list:
-        print(i)
-        i = (i[0]*i[1])
-        return i
-
-
-def largest_product(new_list):
+    new class to complete challenge
```

Comment:
The use of a class for this is not exactly fitting this is more of a function than an object.